core-libs/setup/ssr/optimized-engine/optimized-ssr-engine.ts

```diff
@@ -78,7 +77,7 @@ export class OptimizedSsrEngine {
           ...ssrOptions,
         }
       : undefined;
-    this.logger = this.initLogger(this.ssrOptions);
```

Comment:
`could:` After rethinking it once more, I think we can cast the type to `ExpressServerLogger` as we're sure the value won't be `undefined`

Reply:
done